src/backend/distributed/connection/remote_commands.c

```diff
@@ -52,7 +54,7 @@ ForgetResults(MultiConnection *connection)
 	while (true)
 	{
 		PGresult *result = NULL;
-		result = PQgetResult(connection->pgConn);
+		result = GetRemoteCommandResult(connection, false);
```

Comment:
> Done. But god, I hate that policy.

Like many such policies, I feel this is compensating for poor tools. If I see a `false` there, a hover affordance can show me the name of the variable in the function prototype. I'm certain any other well-configured editor has something similar.